assets/tests/hooks/usePersistedStateReducer.test.ts

```diff
@@ -237,15 +236,14 @@ describe("usePersistedStateReducer", () => {
 
   test("sends updated route tabs to backend on changes", () => {
     const routeTab = routeTabFactory.build({
-      id: "1",
       isCurrentTab: true,
       selectedRouteIds: [],
       ladderDirections: {},
       ladderCrowdingToggles: {},
       ordering: 0,
     })
-    ;(putRouteTabs as jest.Mock).mockImplementationOnce(() =>
-      instantPromise([routeTab])
+    ;(putRouteTabs as jest.Mock).mockImplementationOnce(
+      () => new Promise(jest.fn())
```

Comment:
**question:** should this still use `instantPromise` but instead return a response object with `ok: true` so that we hit line 72 of `usePersistedStateReducer.ts`?

Reply:
actually `instantPromise` won't work because of the `catch`, so a fake promise it is